### DISCUSSION TITLE
Xiaomi Smart Pet Food Feeder & Fountain products

### DISCUSSION THREAD

**Discussion by Antoinekl1** [rytilahti/python-miio#1612] [General]
Hi

Do you know if the 2 new Xiaomi Smart Pet Food Feeder & Fountain products will be integrated soon?

thanks

**Comment by rytilahti** [comment]
If they are newer than two-three years, the genericmiot integration should allow controlling them: #1581
There is also a WIP PR for some type of feeder here, but it was never finished: #1253

**Comment by Antoinekl1** [comment]
Hi make this test for the Smart Pet Fountain

WARNING:miio.device:Found an unsupported model 'mmgg.pet_waterer.wi11' for class 'PetWaterDispenser'. If this is working for you, please open an issue at https://github.com/rytilahti/python-miio/
WARNING:miio.miot_device:Unable to find mapping for mmgg.pet_waterer.wi11, falling back to mmgg.pet_waterer.s1
On: 0
Mode: OperatingMode.Smart
LED on: True
Lid up: False
No water: False
Time without water: 0:00:00
Pump blocked: False
Error detected: False
Days before cleaning left: 1 day, 0:00:00
Cotton filter live left: 0:00:00
Sponge filter live left: 0:00:00
Location: null
Timezone: 127
